x-pack/test/functional/page_objects/tag_management_page.ts

```diff
@@ -57,8 +58,14 @@ class TagModal extends FtrService {
     }
     if (fields.color !== undefined) {
       await this.testSubjects.setValue('~createModalField-color', fields.color);
-      // Wait for the popover to be closable before moving to the next input
-      await new Promise((res) => setTimeout(res, 200));
+      // Close the popover before moving to the next input, as it can get in the way of interacting with other elements
+      await this.testSubjects.existOrFail('euiSaturation');
+      await this.retry.try(async () => {
+        if (await this.testSubjects.exists('euiSaturation', { timeout: 10 })) {
```

Comment:
why it's necessary if you already checked `euiSaturation` exists on L62?

Reply:
It's for any possible retries. If the block retries, but the `euiSaturation` is now gone, it can proceed.